internal/usecase/handlers.go

```diff
@@ -0,0 +1,53 @@
+package usecase
+
+import (
+	r "address-book/internal/usecase/repository"
+
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+const (
+	NotFoundResponse = "Address by this param not found"
+	StatusOKResponse = "OK"
+)
+
+type Handler struct{}
+
+func (h Handler) CreateField(c *gin.Context, repo *r.Repository) {
+	repo.AddField(
+		c.PostForm("name"),
+		c.PostForm("address"),
+		c.PostForm("phone"),
+	)
+	c.JSON(http.StatusOK, StatusOKResponse)
+}
+
+func (h Handler) ReadField(c *gin.Context, repo *r.Repository) {
+	if i, ok := repo.FindField(c.PostForm("param")); ok {
+		c.JSON(http.StatusOK, repo.GetItem(i))
+	}
+	c.JSON(http.StatusNotFound, NotFoundResponse)
+}
+
+func (h Handler) UpdateField(c *gin.Context, repo *r.Repository) {
+	if i, ok := repo.FindField(c.PostForm("param")); ok {
+		repo.DeleteField(i)
+		repo.AddField(
+			c.PostForm("name"),
+			c.PostForm("address"),
+			c.PostForm("phone"),
+		)
```

Comment:
Actually, the update is not the same operation as delete and create the new item